Sky Atlas/Sky Atlas.html

```diff
@@ -24712,39 +24759,69 @@ <h1>Agent Scope Database</h1>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>The Root Edit process begins with a SPK token holder submitting a proposal through the Powerhouse system containing a draft Artifact Edit Proposal. A SPK token holder must hold at least 1% of the circulating token supply to submit a proposal. The proposal must also be posted on the Sky Forum under the "Spark" tag.</td>
+          <td><p>The Root Edit process begins with a SPK token holder submitting a proposal through the Powerhouse system containing a draft Artifact Edit Proposal. A SPK token holder must hold at least 1% of the total token supply to submit a proposal. The proposal must also be posted on the Sky Forum under the “Spark” tag.</p></td>
```

Comment:
1. nit: You don't need p tags for a single line entry
2. avoid changing format of quotations.

```suggestion
          <td>The Root Edit process begins with a SPK token holder submitting a proposal through the Powerhouse system containing a draft Artifact Edit Proposal. A SPK token holder must hold at least 1% of the total token supply to submit a proposal. The proposal must also be posted on the Sky Forum under the "Spark" tag.</td>
```